src/translations/en_US.js

```diff
@@ -24,6 +24,11 @@ module.exports = {
   paypalAccountTokenizationFailedError: 'Something went wrong adding the PayPal account. Please try again.',
   paypalFlowFailedError: 'Something went wrong connecting to PayPal. Please try again.',
   paypalTokenizationRequestActiveError: 'PayPal payment authorization is already in progress.',
+  applePayAccountTokenizationFailedError: 'Something went wrong with Apple Pay. Please try again.',
+  applePayActiveCardError: 'Add a supported card to your Apple Pay wallet.',
+  applePayBrowserNotSupported: 'Browser does not support Apple Pay.',
+  applePayFlowFailedError: 'Something went wrong initializing Apple Pay. Please try again.',
+  applePayTokenizationRequestActiveError: 'Apple Pay is already in progress.',
```

Comment:
Should it be "Apple Pay payment is already in progress..." or something like the PayPal error? Also can this error actually happen since the flow is part of the browser?

Reply:
Looks like this error is not used so I'm going to go ahead and remove it.